packages/opencensus-web-types/src/exporters/types.ts

```diff
@@ -56,6 +56,9 @@ export interface StatsEventListener {
    * batched data to backend.
    */
   start(): void;
+
+  /** Stops the exporter. */
+  stop(): void;
```

Comment:
Did you need to implement this somewhere?

Reply:
As we are not using Stats, we are not actually implementing this interface.